### ISSUE TITLE
couldn't find func import: env._emscripten_get_heap_size

### ISSUE THREAD

**Issue by geloizi** [rianhunter/wasmjit#19]
I got the next error with the most recent version of wasmjit:

```
emcc -o selfpipe.js src/wasmjit_examples/selfpipe.c
./wasmjit selfpipe.wasm                                                                                                                                                                     
failed to instantiate module: couldn't find func import: env._emscripten_get_heap_size [] -> [I32,]
```

**Comment by rianhunter** [comment]
What version of emscripten are you using?

**Comment by geloizi** [comment]
```
 emcc --version                                                                                                                                                                            emcc (Emscripten gcc/clang-like replacement) 1.38.25 (commit ea5d631a5446632e195765d89a53ead71cd6de45)
Copyright (C) 2014 the Emscripten authors (see AUTHORS.txt)
This is free and open source software under the MIT license.
There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

**Comment by rianhunter** [comment]
ah yeah, there were some important changes made to emscripten in version 1.38.23 and up. i haven't yet updated the public wasmjit repo to support those changes. if you want a short term solution, use emscripten version 1.38.22

**Comment by rianhunter** [comment]
fixed in bdc30171001310121af807af01f430232640459a
